pkg/core/resources/model/rest/unmarshaller.go

```diff
@@ -84,25 +81,6 @@ func (u *unmarshaler) Unmarshal(bytes []byte) (Resource, error) {
 	return restResource, nil
 }
 
-func (u *unmarshaler) rawSchemaValidation(bytes []byte, schema *spec.Schema) error {
```

Comment:
Also unrelated to MeshAccessLog isn't it? No reason to block the PR but really should be in its own PR for visibility/understanding/traceability

Reply:
Right, it was a small problem with validation that I noticed after the changes in MeshAcessLog. Should've opened another PR